src/Core/BaseKernel.php

```diff
@@ -469,4 +474,27 @@ public function setThemeResourceLoader($themeResourceLoader)
         $this->themeResourceLoader = $themeResourceLoader;
         return $this;
     }
+
+    /**
+     * Validate configuration of the application is in a good state, ready for use.
+     *
+     * This method can be used to warn developers of any misconfiguration, or configuration
+     * which is missing but should be set according to best practice.
+     *
+     * In some cases, this could be used to halt execution if configuration critical to operation
+     * has not been set.
+     */
+    protected function validateConfiguration(): void
```

Comment:
I chose to do this as a protected method because the fact that `bootConfigs()` is called from `BaseKernel` is an implementation detail of that kernel - other kernels could be implemented to intentionally avoid calling that method, so this allows them a way to still perform this validation.

This also provides a clear place for this sort of validation to be done for other config in the future, if we find a need to do so.